settings.gradle.kts

```diff
@@ -50,6 +50,8 @@ include(
     ":demos:appyx-navigation:android",
     ":demos:appyx-navigation:desktop",
     ":demos:appyx-navigation:web",
+    ":demos:appyx-navigation:ios",
+    ":demos:appyx-interactions:ios",
```

Comment:
I'd rather we move the second line to the interactions block (line 42 onwards) + both of them in alphabetical order